windowing_library.py

```diff
@@ -0,0 +1,71 @@
+import pickle
+import matplotlib.pyplot as plt
+import numpy as np
+import skimage 
+from scipy import signal
+from scipy.spatial import procrustes
+from skimage.util.shape import view_as_windows
+import statistics
+
+# make it usable for all rows and make one function you call first for phase 1 and 2 then a second main function for phase 3
+
+def create_windows(matrix, rows, window_size, stride, mod_num):
+  x = matrix['x']
+  y = matrix['y']
+  z = matrix['z']
+  nanos = matrix['nanos'] 
+  np_data = np.array([x,y,z,nanos], 'd')
+
+  windowed_data = view_as_windows(np_data, (rows, window_size), step=stride)
+  return windowed_data
+
+def find_initial_lift_times(lift_windows, rows):
+  start = lift_windows[0][rows-1][0]
+  end = lift_windows[-1][-1][-1]
+  return start, end
+
+def find_lift_windows(windowed_data):
+  variance=[]
+  lift_windows =[]
+  windows = windowed_data[0]
+  for i in range(len(windows)):
+	x = windows[i][0]
+	y = windows[i][1]
+	z = windows[i][2]
+	nanos = windows[i][3]
```

Comment:
remove variables that you don't use. It'll make your code more concise and easier to read.